gstcefsrc.cc

```diff
@@ -525,66 +654,111 @@ run_cef (GstCefSrc *src)
     /* unblock start () */
     g_mutex_lock (&init_lock);
     cef_inited = TRUE;
-    g_cond_signal(&init_cond);
+    g_cond_broadcast (&init_cond);
     g_mutex_unlock (&init_lock);
 
-    goto done;
+    return NULL;
```

Comment:
nit: I like single exit points functions if you don't mind keeping it that way :)